Formula/now-cli.rb

```diff
@@ -0,0 +1,15 @@
+class NowCli < Formula
+  desc "The command-line interface for Now"
+  homepage "https://zeit.co/now"
+  url "https://github.com/zeit/now-cli/releases/download/15.8.0/now-macos.gz"
```

Comment:
Files can not be OS specific and ideally should be build from source. https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
I've changed it to build from source, please help to review.